src/components/common/state-machine/state-machine.ts

```diff
@@ -146,6 +150,10 @@ const authStateMachine = createMachine(
               cond: "isIdentityRequired",
             },
             { target: [PATH_NAMES.SIGN_IN_OR_CREATE] },
+            {
```

Comment:
I don't think we want this condition - if there's no existing session, they should just be going through the normal sign in or create path I think. There are also no tests for this case.